DOC: Simplify bug report template (#7329)
Slicer/Slicer: .github/ISSUE_TEMPLATE/bug_report.md

## Steps to reproduce

<!--
    Generally, Slicer developers can only fix those issues that they can reproduce on their computers.
    
    To achieve this, please:
    * Describe in this section what you did, what you expected to happen, and what happened instead.
    * Attach a few screenshots if possible.
    * Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing).
  

>     If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org

Author: ```suggestion
    If the problem cannot be reproduced by using the graphical user interface but only by running custom Python or C++ code then please create a short self-contained example as described at http://sscce.org
```

  
    To describe "Actual behavior" and "Expected behavior" you may use the following format:
    1. Do A
    2. Do B => Slicer does Y - OK

>     3. Do C => Slicer does Z - WRONG -> Slicer should do W instead

Author: ```suggestion
    3. Do C => Slicer does Z => ERROR: Slicer should do W instead
```

    4. Do D => Slicer does Z - OK

>     5. Do B => Slicer does Y - WRONG -> Slicer should do X instead

Author: ```suggestion
    5. Do B => Slicer does Y => ERROR: Slicer should do X instead
```

## Steps to reproduce

> <!-- Generally, Slicer developers can only fix those issues that they can reproduce on their computers. To achieve this, please describe in this section what you did, what you expected to happen, and what happened instead. Attach a few screenshots if possible. Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing). If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org -->

Reviewer: ```suggestion
<!--
    Generally, Slicer developers can only fix those issues that they can reproduce on their computers.
    
    To achieve this, please:
    * Describe in this section what you did, what you expected to happen, and what happened instead.
    * Attach a few screenshots if possible.
    * Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing).
  
  If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org
  
  To describe "Actual behavior" and "Expected behavior" you would use the following format:
  1. Do A
  2. Do B => Slicer does Y - OK
  3. Do C => Slicer does Z - WRONG -> Slicer should do W instead
  4. Do D => Slicer does Z - OK
  5. Do B => Slicer does Y - WRONG -> Slicer should do X instead
-->
```
